### PR TITLE
Final changes before OSM

### PR DESCRIPTION
**Pull request by jbusecke**
@dcherian, can you read over this and make changes where you see fit?

Ill check on the binder performance now.

### CODE REVIEW

**Comment by jbusecke** [comment]
While you are at it, could you replace
```python
sorted(cat.df['source_id'].unique())
```

with 

```python
sorted(col.df['source_id'].unique())
```

**Comment by jbusecke** [comment]
Ooof this is really slow...

**Comment by jbusecke** [comment]
I am trying the pangeo AWS binder...

File: getting_started.ipynb [hunk 1]
```diff
@@ -5,15 +5,17 @@
    "id": "bd00bd98-9b7e-4437-8518-90d25e69a00e",
```
Comment (jbusecke) [inline getting_started.ipynb:5]: <p>add link (https://pangeo.io/cloud.html) to pangeo deployments?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/tutorial_osm22_cmip6/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Comment by jbusecke** [comment]
No dice: https://aws-uswest2-binder.pangeo.io/v2/gh/jbusecke/tutorial_osm22_cmip6/some_more_changes?filepath=getting_started.ipynb

This somehow fails, not sure if there is a way to debug this until tomorrow. 

I think we should then remove the timeseries cell (it is at 1% now after 11 minutes)

**Comment by jbusecke** [comment]
This is just one model: 
image

wahhhh

**Comment by dcherian** [comment]
Yes delete the cell.

File: getting_started.ipynb (hunk 1 above)
Comment (jbusecke) [inline getting_started.ipynb:5]: <p>change <code>variable for  You</code> to <code>variable. You</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/tutorial_osm22_cmip6/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

**Comment by jbusecke** [comment]
Love the changes. Could you maybe clear all the outputs, so the participants dont see all the results? Optional to me.

**Pull request merged**